controller/public/index.html

```diff
@@ -0,0 +1,1010 @@
+<!doctype html>
```

Comment:
Oh I see, we have the generated API in the .git repository.

I guess it's fine for now, but it's perhaps better to not store generated files in the .git repository if not needed.